core/client/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -121,26 +118,26 @@ public void close() throws IOException {
    *         a folder
    */
   @Override
-  public FSDataOutputStream create(Path cPath, FsPermission permission, boolean overwrite,
+  public FSDataOutputStream create(Path path, FsPermission permission, boolean overwrite,
       int bufferSize, short replication, long blockSize, Progressable progress) throws IOException {
-    LOG.info("create({}, {}, {}, {}, {}, {}, {})", cPath, permission, overwrite, bufferSize,
+    LOG.info("create({}, {}, {}, {}, {}, {}, {})", path, permission, overwrite, bufferSize,
         replication, blockSize, progress);
     if (mStatistics != null) {
       mStatistics.incrementWriteOps(1);
     }
 
     // Check whether the file already exists, and delete it if overwrite is true
-    AlluxioURI path = new AlluxioURI(HadoopUtils.getPathWithoutScheme(cPath));
+    AlluxioURI uri = new AlluxioURI(HadoopUtils.getPathWithoutScheme(path));
     try {
-      if (mFileSystem.exists(path)) {
+      if (mFileSystem.exists(uri)) {
         if (!overwrite) {
-          throw new IOException(ExceptionMessage.FILE_ALREADY_EXISTS.getMessage(cPath.toString()));
+          throw new IOException(ExceptionMessage.FILE_ALREADY_EXISTS.getMessage(path.toString()));
```

Comment:
Not something this PR introduces, but since we are changing it, should path be `uri` to be consistent with the above logging for `append`?